### PR TITLE
Update AlexVCS.ts

### PR DESCRIPTION
**Pull request by AlexVCS**
## Linked Issue

https://github.com/Virtual-Coffee/virtualcoffee.io/issues/995

## Description
Added my location to my member profile with latitude and longitude.

## Methodology

Looked it up and pasted it in.

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

File: members/members/AlexVCS.ts [hunk 1]
```diff
@@ -42,5 +42,10 @@ export const AlexVCS: MemberObject = {
 		// { type: 'hashnode', username: 'yourUserName' },
 		// { type: 'website', url: 'https://virtualcoffee.io', title: 'Title of link' },
 	],
-	badges: ['Hacktoberfest2022'],
+	badges: ['Hacktoberfest2022'], ['Hacktoberfest2023'],
```
Comment (danieltott) [inline members/members/AlexVCS.ts]: Looks like we need to fix up the syntax here

```suggestion
	badges: ['Hacktoberfest2022' 'Hacktoberfest2023'],
```

File: members/members/AlexVCS.ts [hunk 2]
```diff
@@ -42,5 +42,10 @@ export const AlexVCS: MemberObject = {
...
+	location: {
+  latitute: 27.664827,
```
Comment (danieltott) [inline members/members/AlexVCS.ts]: Change this to `latitude`:

```suggestion
  latitude: 27.664827,
```

**Review by danieltott — CHANGES_REQUESTED** [review]
Thank you @AlexVCS - got a couple small tweaks for you.

**Review by AlexVCS — COMMENTED** [review]
thanks I fixed it

**Review by danieltott — APPROVED** [review]
Looks great!

**Pull request merged**